data-serving/data-service/src/model/case.ts

```diff
@@ -135,7 +135,7 @@ const caseSchema = new mongoose.Schema(
                         type: String,
                         required: 'Enter a name for the event',
                     },
-                    date: dateRangeSchema,
+                    dateRange: dateRangeSchema,
```

Comment:
thanks for fixing! slowly working my way through refactoring + adding tests

Reply:
nw!